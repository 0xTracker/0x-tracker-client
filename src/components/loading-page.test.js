import { render } from 'react-testing-library';
import React from 'react';

import LoadingPage from './loading-page';

it('should render loading page', () => {
  const { container } = render(<LoadingPage />);

  expect(container.firstChild).toMatchSnapshot();
});
