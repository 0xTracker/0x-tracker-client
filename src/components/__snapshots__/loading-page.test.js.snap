// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render loading page 1`] = `
<div
  class="page-layout__StyledPageLayout-cls0ob-1 erAgYD"
>
  <div
    class="container__StyledContainer-cj53s7-0 fLJuGb page-layout__PageBody-cls0ob-0 kPXbwD container-fluid"
  >
    <div
      style="fill: currentColor; height: 50px; width: 50px;"
    >
      <svg
        viewBox="0 0 32 32"
        xmlns="http://www.w3.org/2000/svg"
      >
        
  
        <path
          d="M16 0 A16 16 0 0 0 16 32 A16 16 0 0 0 16 0 M16 4 A12 12 0 0 1 16 28 A12 12 0 0 1 16 4"
          opacity=".25"
        />
        
  
        <path
          d="M16 0 A16 16 0 0 1 32 16 L28 16 A12 12 0 0 0 16 4z"
        >
          
    
          <animatetransform
            attributeName="transform"
            dur="0.8s"
            from="0 16 16"
            repeatCount="indefinite"
            to="360 16 16"
            type="rotate"
          />
          
  
        </path>
        

      </svg>
      

    </div>
  </div>
</div>
`;
